docs/study-def-dates.md

```diff
@@ -5,9 +5,10 @@ In study definitions, dates are described in `"YYYY-MM-DD"` format; so, for exam
 
 Most variable extractor functions have arguments for specifying the date range over which you want to retrieve information.
 Most commonly this is `on_or_before=`, `on_or_after=`, or `between=` (see the [variable reference](study-def-variables.md) for full documentation).
-You must use at most one.
+You must use at most one. 
```

Comment:
```suggestion
At most, you can use one of these functions.
```

Reply:
Thinking about it I don't actually think this it's true that you can't use more than one - you can use both `on_or_before=` and `on_or_after=`, you just can't combine them with `between`. Is that right?

Reply:
HAve made a further change to this - how does it look?